app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,69 @@
+class PasswordResetsController < ApplicationController
+  before_action :load_user, :check_user_actived, only: %i(edit create update)
+  before_action :valid_user, :check_expiration, only: %i(edit update)
+  before_action :check_password_presence, only: :update
+
+  def new; end
+
+  def edit; end
+
+  def create
+    @user.create_reset_digest
+    @user.send_password_reset_email
+    flash[:info] = t("password_reset.send_mail")
+    redirect_to login_path
+  end
+
+  def update
+    if @user.update(user_params)
```

Comment:
```suggestion
    if @user.update(user_params.merge(reset_digest: nil))
```
Thử thực hiện line 20 trong line này luôn.

Reply:
[Uploading Screencast from 08-03-2024 07:59:28.webm…]()